DictionaryTypes.txt

```diff
@@ -274,6 +277,7 @@ enum(ch_type) "Type of the channel"
     dipole_wave    1000		     "Dipole time curve"
     goodness_fit   1001		     "Goodness of fit"
     fnirs	   1100		     "Functional near-infrared spectroscopy"
+    eyetrack   1200          "Eye-tracking"
```

Comment:
Hey @dominikwelke can we update this file to be in line with the code in `mne/io/constants.py` on our PR? 

Basically, PR's on adding skin temp and galvonic skin response channels got merged after you started the eyetracking work. 

So Eyetrack channels got bumped from 1200 to 1400, because temperature channels are in the 1200's, gsr in the 1300's:

From `mne/io/constants.py`:
```
FIFF.FIFFV_FNIRS_CH     = 1100  # Functional near-infrared spectroscopy
FIFF.FIFFV_TEMPERATURE_CH = 1200  # Functional near-infrared spectroscopy
FIFF.FIFFV_GALVANIC_CH  = 1300  # Galvanic skin response
FIFF.FIFFV_EYETRACK_CH  = 1400  # Eye-tracking
```

Once we update this, `mne/io/tests/test_constants.py` should pass :slightly_smiling_face:

Reply:
done @scott-huberty 

here's the commit if you want to update it in your pr branch ([mne/io/tests/test_constants.py](https://github.com/mne-tools/mne-python/pull/11152/files#diff-d545e84fd98e16f320b83c21e8c35a67ed64d378ded8fb35d3734e9f2196b53e)):

e300d7a81b3b7459da8ffdcc04d96c6a95f4e841

Reply:
awesome thanks. And now there are no conflicts between this branch and the base branch. 2 birds 1 stone..